maestro-orchestra-models/src/main/java/maestro/orchestra/Commands.kt

```diff
@@ -327,21 +327,43 @@ data class AssertConditionCommand(
     }
 }
 
-data class InputTextCommand(
+data class InputTextCommandV1(
     val text: String
 ) : Command {
 
     override fun description(): String {
         return "Input text $text"
     }
 
-    override fun evaluateScripts(jsEngine: JsEngine): InputTextCommand {
+    override fun evaluateScripts(jsEngine: JsEngine): InputTextCommandV1 {
         return copy(
             text = text.evaluateScripts(jsEngine)
         )
     }
 }
 
+data class InputTextCommandV2(
+    val text: String,
+    val accessibilityText: String? = null,
+    val id: String? = null,
+    val point: String? = null,
+    val pasteTitle: String? = null
+) : Command {
+    override fun description(): String {
+        return "Input text v2 $text"
```

Comment:
This is being presented to users, so maybe we should remove the `v2` part? We could also update it to add something indicating that it is inputting text to an element directly, thus indicating it is the v2 version.

```suggestion
        return "Input text $text"
```